runtime/kubernetes.go

```diff
@@ -607,6 +1104,39 @@ func (rt Kubernetes) CreateSecret(space string, name string, data string, mimety
 	return &secret, nil
 }
 
+func (rt Kubernetes)  CopySecret(secretName string, fromNamespace string, toNamespace string) (error) {
```

Comment:
Rather than need a reference to vault to get the image pull secret, this allow us to copy it out from `akkeris-system`